docs/advanced_topics/icons.md

```diff
@@ -30,9 +30,9 @@ Draw or download an icon and save it in a template folder:
 
 The `svg` tag should:
 
--   Set the `id="icon-<name>"` attribute, icons are referenced by this name
--   Set the `xmlns="http://www.w3.org/2000/svg"` attribute
--   Set the `viewBox="..."` attribute
+-   Set the `id="icon-<name>"` attribute, icons are referenced by this name.
+-   Set the `xmlns="http://www.w3.org/2000/svg"` attribute.
+-   Set the `viewBox="..."` attribute, and no `width` and `height` attributes.
```

Comment:
Noticed this on the `sliders` icon. This isn’t that big of a problem, just done for consistency.